tasks/verdaccio.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+custom_registry_url=http://localhost:4873
+original_npm_registry_url=`npm get registry`
+original_yarn_registry_url=`yarn config get registry`
+
+function startVerdaccio {
+  # Start local registry
+  tmp_registry_log=`mktemp`
+  (cd && nohup npx https://createreactapp.blob.core.windows.net/lib/verdaccio-4.0.0-alpha.5.tgz -c $1 &>$tmp_registry_log &)
```

Comment:
Using a patched version of Verdaccio (for now) that allows us to configure HTTP Agent options (`request/agentOptions`). This is necessary to work around an Azure VM specific issue caused when too many sockets are opened. See https://docs.microsoft.com/en-us/azure/app-service/app-service-web-nodejs-best-practices-and-troubleshoot-guide#my-node-application-is-making-excessive-outbound-calls